execution_workflows/APA-Scan/workflow/envs/APA-Scan.Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Template Dockerfile with python and pandas.
```

Comment:
See other comments on how to get the APA-Scan code inside the container as well, then update the Snakefile to use Docker rather than Conda for cloud execution